pkg/api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -40,6 +40,73 @@ func (in *AlternatorSpec) DeepCopy() *AlternatorSpec {
 	return out
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
```

Comment:
Do we need to have autogenerated code in the repository?

Reply:
Keeping it here makes build process easier. It also helps with keeping builds reproducible over time.

Reply:
Why would the build not be reproducible? Is the generator non deterministic? Is that intentional?

Reply:
This one is deterministic (I hope so), but I can't tell that it will work the same in newer versions. Keeping autogenerated files in repository protects us from these kind of changes with low price.
It also helps with development process because it's easier to see how your code changes affects autogenerated ones. 
I don't have a good idea how to get rid of it, moving them to separate repository doesn't make sense. Not keeping it repository, breaks Go Module immutability.